### DISCUSSION TITLE
Integration with captum for explainability

### DISCUSSION THREAD

**Discussion by akashlp27** [open-mmlab/mmclassification#1090] [Ideas] [answered]
Since mmclassification is also built with Pytorch I found it would be interesting if we could get explainable AI to be included too along GradCam since, Captum can give better explainability  .

It will be a addon to visualize how the model learns and also the explainability of the predictions or the model outcome. 

Attached is the official GitHub page of captum. 
https://github.com/pytorch/captum

**Comment by Ezra-Yu** [comment, marked as answer]
Thank you for your suggestion, We have support vis-CAM refer to [doc](https://mmclassification.readthedocs.io/en/latest/tools/visualization.html#class-activation-map-visualization.)

As for [Model Interpretation](https://captum.ai/tutorials/Resnet_TorchVision_Interpret), We will investigate this feature.

**Comment by akashlp27** [comment]
It will be really great, thank you....!
